### PR TITLE
Describe Authorization scopes

### PR DESCRIPTION
**Pull request by bajtos**
Documentation for the new feature to be added by https://github.com/strongloop/loopback/pull/3313.

Connect to strongloop/loopback#382

### CODE REVIEW

**Comment by bajtos** [comment]
(This pull request must not be landed before the change in LoopBack is finished.)

**Review by superkhau — APPROVED** [review]

**Comment by crandmck** [comment]
Actually, the doc PR should not be landed until the code is landed AND published to npm.

**Comment by bajtos** [comment]
I have reworked the pull request to match what has been landed by strongloop/loopback#3313.

@ebarault @crandmck could you please review again?

**Review by crandmck — CHANGES_REQUESTED** [review]
I went ahead and made wording and grammar edits directly in the branch, since that seemed most efficient.

IIUC, we should also update https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#methods since this feature will now be implemented.  Is that right?
If so, please add to this PR--or if you prefer I can take a stab at it next week.
Again, if I'm understanding, this reference info should largely duplicate what's in https://loopback.io/doc/en/lb3/Remote-methods.html#options.   If so, then instead of 
actually duplicating these tables, we can move them into the `_includes/content` and
single-source them so we only have to maintain them there.

File: pages/en/lb3/Controlling-data-access.md
Comment (bajtos) [inline pages/en/lb3/Controlling-data-access.md]: @crandmck 
`accessScopes` is not a model property, it is a new property in remote method metadata. My proposal:

```diff
- The new model property `accessScopes` defines scopes as part of remoting metadata.
+ The new setting `accessScopes` defines scopes as part of remoting metadata.
```

**Comment by bajtos** [comment]
@crandmck thank you for the edits.

> https://loopback.io/doc/en/lb3/Remote-methods.html#options

Good catch, `accessScopes` should be added to that table. I'll do that ASAP.

> IIUC, we should also update https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#methods since this feature will now be implemented. Is that right?

I am confused. The current doc pages says that remote methods cannot be configured via JSON files yet, but that hasn't been true for months. So yes, we should fix that section and I think that fix is independent of the changes I am making here. I am proposing to leave that out of scope of this pull request.

> Again, if I'm understanding, this reference info should largely duplicate what's in https://loopback.io/doc/en/lb3/Remote-methods.html#options. If so, then instead of
actually duplicating these tables, we can move them into the _includes/content and
single-source them so we only have to maintain them there.

Ditto, I am proposing to leave `_includes/content` out of scope of this patch.

**Comment by bajtos** [comment]
Updated, PTAL.

File: pages/en/lb3/Controlling-data-access.md
Comment (crandmck) [reply pages/en/lb3/Controlling-data-access.md]: IIUC, it's a value set in the model JSON file, which is what I meant by a "model property".  Apparently that is confusing, but "setting" doesn't really convey any specific meaning.  

Perhaps it would be clearer to say 
> The `methods.accessScopes` property in the model JSON file defines...

**Comment by crandmck** [comment]
> The current doc pages says that remote methods cannot be configured via JSON files yet, but that hasn't been true for months. So yes, we should fix that section and I think that fix is independent of the changes I am making here. I am proposing to leave that out of scope of this pull request.

No one told me, created an issue, or opened a doc PR, so apparently it was not documented (unless I'm missing something).  It's closely related to this PR, but we can do it separately if you prefer.
Added #352.

With [one small edit](https://github.com/strongloop/loopback.io/pull/338#discussion_r110962379) noted above I think this is ready to land, WHEN the change is published to npm.  We shouldn't land this until that time, though.

**Review by crandmck — APPROVED** [review]

**Comment by crandmck** [comment]
Note: https://github.com/strongloop/loopback.io/commit/c3059946e7673214dc97f2487aa09ed83414c849 will require this to be rebased, because I added the doc for remote method config via JSON per #352.

File: pages/en/lb3/Controlling-data-access.md
Comment (bajtos) [reply pages/en/lb3/Controlling-data-access.md]: Done in ac92730

**Comment by bajtos** [comment]
> Note: c305994 will require this to be rebased, because I added the doc for remote method config via JSON per #352.

Rebased, I moved the newly-added remote method option "accessScopes" to the new place you created.

>  I think this is ready to land, WHEN the change is published to npm. We shouldn't land this until that time, though.

Sounds good. I'd like to land https://github.com/strongloop/loopback/pull/3350 before making a new loopback 3.x release.

**Comment by bajtos** [comment]
All required changes were landed and released in `loopback@3.7.0`. I'm going to rebase and land this patch.

**Pull request merged**
